docs/references/mqtt-namespace.md

```diff
@@ -828,6 +829,8 @@ The container JSON message is comprised of the following elements:
 
 * Type: denotes the type of inventory payload
 
+* State: describes the container's current state
```

Comment:
Can you expand on this? What kind of state and what do they mean? A bullet list would be enough..